applications/minotari_ledger_wallet/comms/src/accessor_methods.rs

```diff
@@ -126,25 +126,25 @@ fn verify() -> Result<(), LedgerDeviceError> {
             Ok(public_key) => {
                 if !signature.verify(&public_key, nonce) {
                     return Err(LedgerDeviceError::Processing(
-                        "'Minotari Wallet' application could not create a valid signature. Please update the firmware \
-                         on your device."
+                        "Error 1: 'Minotari Wallet' application could not create a valid signature. Please update the \
+                         firmware on your device."
                             .to_string(),
                     ));
                 }
                 signature
             },
             Err(e) => {
                 return Err(LedgerDeviceError::Processing(format!(
-                    "'Minotari Wallet' application could not retrieve a public key ({:?}). Please update the firmware \
-                     on your device.",
+                    "Error 2: 'Minotari Wallet' application could not retrieve a public key ({:?}). Please update the \
```

Comment:
NIT: The wording `Error 2` etc could be confusing when the user receives a single error and wonders why it is claiming it's the 2nd or 3rd error.

Reply:
It made my life a lot easier when I had to debug it earlier today, instead of trying to match the error text visually.

Reply:
yeah I kinda agree with the nit here